dipy/workflows/reconst.py

```diff
@@ -979,6 +985,8 @@ def run(self, input_files, bvalues_files, bvectors_files, mask_files,
                        and perfusion in intravoxel incoherent motion MR
                        imaging." Radiology 168.2 (1988): 497-505.
         """
+        if save_metrics is None:
+            save_metrics = []
```

Comment:
`save_metrics = save_metrics or []` instead

Reply:
Done in f49517e.